BitTornado/Meta/Info.py

```diff
@@ -230,6 +230,8 @@ def __init__(self, name, size=None,
             self['files'] = []
             self['length'] = size
 
+        self['private'] = params['private'] if 'private' in params else False
```

Comment:
If you want to make it an optional field:

    if 'private' in params:
        self['private'] = params['private']

Otherwise, a quick way of doing this:

    self['private'] = params.pop('private', False)